src/utils/jsonp/qase-jsonp.ts

```diff
@@ -0,0 +1,13 @@
+import {getJsonpResolver} from './jsonp-resolvers';
+
+export function qaseJsonp(data: any): void {
+    const callbackId: string | undefined = document.currentScript?.dataset?.callbackId;
+    if (!callbackId) {
+        throw new Error('No callbackId for jsonp data was found!');
+    }
+    const resolve = getJsonpResolver(callbackId);
+    if(!resolve) {
+        throw new Error('No resolver was found!')
```

Comment:
You have prettier in the project but seemingly it is not set up in your IDE.
I'd recommend doing it and running `npx prettier 'src/**/*.{ts,tsx}' --write` to auto-fix current problems.
Notice that the current `.prettierrc.json` dictates having 2-spaces indent and eliminating semicolons

Reply:
yes, for sure